### PR TITLE
expand percolator and dda documentation

### PR DESCRIPTION
**Pull request by timosachsenberg**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by ypriverol — APPROVED** [review]

File: docs/dda.rst [hunk 1]
```diff
@@ -1,6 +1,26 @@
 Data-dependent acquisition (DDA) quantification
 ===============================================
 
+In bottom-up proteomics, DDA is used to collect tandem MS spectra for identification
```
Comment (jpfeuffer) [inline docs/dda.rst:4]: The first sentence sounds wrong somehow.

**Review by jpfeuffer — COMMENTED** [review]
Do you want to extend later? Cause I think  Percolator could use some extended tips. E.g. on roughly which features are used for which search engines/consensus. When it can go wrong. Alternatives, Advantages, disadvantages. Which features of it we support, which not.
Plfq similarly.

File: docs/dda.rst (hunk 1 above)
Comment (timosachsenberg) [reply docs/dda.rst:4]: "the DDA method is used"?

**Comment by timosachsenberg** [comment]
Yes will extend it

**Comment by jpfeuffer** [comment]
Please merge yourself or write "ready to merge" whenever you want to test a PR.
You can also test PRs yourself with "make html".

**Pull request merged**